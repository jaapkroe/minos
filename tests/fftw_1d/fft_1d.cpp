#include <iostream>
#include <fstream>
#include <fftw3.h>
#define NMAX 8192
int main()
{
  int i, N=0, M=10;
  double *input = new double [NMAX];
  std::ifstream file("fft_1d.dat");
  while (file >> input[N]) { N++; if(N>=NMAX) { fprintf(stderr,"ERROR: N>NMAX\n"); exit(1); }; };
  if(M>N) M=N;
  fprintf(stderr,"read %d data points, generating FFT with %d q-points\n",N,M);

  fftw_complex *out = fftw_alloc_complex(2*N/1);
  fftw_plan p = fftw_plan_dft_r2c_1d(N,input,out,FFTW_ESTIMATE);
  fftw_execute(p);

  for(i=0;i<M;i++)
    printf("%-4d %12.4g %12.4g\n",i,out[i][0]/N,out[i][1]/N);
}
