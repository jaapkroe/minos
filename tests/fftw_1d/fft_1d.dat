3.72665e-06
9.9295e-06
2.54193e-05
6.25215e-05
0.000147748
0.000335463
0.000731802
0.00153381
0.00308872
0.00597602
0.011109
0.0198411
0.0340475
0.0561348
0.0889216
0.135335
0.197899
0.278037
0.375311
0.486752
0.606531
0.726149
0.83527
0.923116
0.980199
1
0.980199
0.923116
0.83527
0.726149
0.606531
0.486752
0.375311
0.278037
0.197899
0.135335
0.0889216
0.0561348
0.0340475
0.0198411
0.011109
0.00597602
0.00308872
0.00153381
0.000731802
0.000335463
0.000147748
6.25215e-05
2.54193e-05
9.9295e-06
