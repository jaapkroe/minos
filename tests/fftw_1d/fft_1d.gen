0 -6.55706e-05
0.00628319 -6.55548e-05
0.0125664 -6.55076e-05
0.0188496 -6.54287e-05
0.0251327 -6.53183e-05
0.0314159 -6.51761e-05
0.0376991 -6.50021e-05
0.0439823 -6.4796e-05
0.0502655 -6.45578e-05
0.0565487 -6.42871e-05
0.0628319 -6.39839e-05
0.069115 -6.36477e-05
0.0753982 -6.32783e-05
0.0816814 -6.28753e-05
0.0879646 -6.24384e-05
0.0942478 -6.19672e-05
0.100531 -6.14613e-05
0.106814 -6.09201e-05
0.113097 -6.03432e-05
0.119381 -5.973e-05
0.125664 -5.908e-05
0.131947 -5.83925e-05
0.13823 -5.76668e-05
0.144513 -5.69023e-05
0.150796 -5.60982e-05
0.15708 -5.52537e-05
0.163363 -5.43679e-05
0.169646 -5.344e-05
0.175929 -5.2469e-05
0.182212 -5.14539e-05
0.188496 -5.03937e-05
0.194779 -4.92873e-05
0.201062 -4.81334e-05
0.207345 -4.69309e-05
0.213628 -4.56784e-05
0.219911 -4.43747e-05
0.226195 -4.30182e-05
0.232478 -4.16075e-05
0.238761 -4.0141e-05
0.245044 -3.8617e-05
0.251327 -3.70339e-05
0.257611 -3.53898e-05
0.263894 -3.36827e-05
0.270177 -3.19109e-05
0.27646 -3.0072e-05
0.282743 -2.81641e-05
0.289027 -2.61848e-05
0.29531 -2.41317e-05
0.301593 -2.20024e-05
0.307876 -1.97943e-05
0.314159 -1.75047e-05
0.320442 -1.51308e-05
0.326726 -1.26697e-05
0.333009 -1.01182e-05
0.339292 -7.47323e-06
0.345575 -4.73146e-06
0.351858 -1.88942e-06
0.358142 1.05648e-06
0.364425 4.11001e-06
0.370708 7.27505e-06
0.376991 1.05557e-05
0.383274 1.39561e-05
0.389557 1.74807e-05
0.395841 2.1134e-05
0.402124 2.49209e-05
0.408407 2.88461e-05
0.41469 3.29149e-05
0.420973 3.71325e-05
0.427257 4.15045e-05
0.43354 4.60365e-05
0.439823 5.07346e-05
0.446106 5.56049e-05
0.452389 6.06538e-05
0.458673 6.58879e-05
0.464956 7.13141e-05
0.471239 7.69396e-05
0.477522 8.27716e-05
0.483805 8.8818e-05
0.490088 9.50866e-05
0.496372 0.000101586
0.502655 0.000108324
0.508938 0.000115309
0.515221 0.000122552
0.521504 0.000130061
0.527788 0.000137846
0.534071 0.000145918
0.540354 0.000154286
0.546637 0.000162961
0.55292 0.000171956
0.559203 0.000181281
0.565487 0.000190948
0.57177 0.000200969
0.578053 0.000211358
0.584336 0.000222128
0.590619 0.000233293
0.596903 0.000244865
0.603186 0.000256861
0.609469 0.000269295
0.615752 0.000282182
0.622035 0.000295539
0.628319 0.000309382
0.634602 0.000323729
0.640885 0.000338596
0.647168 0.000354003
0.653451 0.000369968
0.659734 0.00038651
0.666018 0.00040365
0.672301 0.000421407
0.678584 0.000439804
0.684867 0.000458862
0.69115 0.000478604
0.697434 0.000499053
0.703717 0.000520233
0.71 0.000542168
0.716283 0.000564885
0.722566 0.00058841
0.728849 0.000612769
0.735133 0.00063799
0.741416 0.000664103
0.747699 0.000691136
0.753982 0.000719121
0.760265 0.000748087
0.766549 0.000778068
0.772832 0.000809097
0.779115 0.000841208
0.785398 0.000874436
0.791681 0.000908816
0.797965 0.000944387
0.804248 0.000981185
0.810531 0.00101925
0.816814 0.00105863
0.823097 0.00109935
0.82938 0.00114146
0.835664 0.00118502
0.841947 0.00123005
0.84823 0.00127661
0.854513 0.00132474
0.860796 0.0013745
0.86708 0.00142593
0.873363 0.00147908
0.879646 0.00153401
0.885929 0.00159078
0.892212 0.00164943
0.898495 0.00171003
0.904779 0.00177263
0.911062 0.00183729
0.917345 0.00190408
0.923628 0.00197306
0.929911 0.00204429
0.936195 0.00211784
0.942478 0.00219377
0.948761 0.00227217
0.955044 0.00235309
0.961327 0.00243662
0.967611 0.00252283
0.973894 0.00261179
0.980177 0.00270359
0.98646 0.0027983
0.992743 0.00289601
0.999026 0.0029968
1.00531 0.00310076
1.01159 0.00320798
1.01788 0.00331855
1.02416 0.00343256
1.03044 0.00355011
1.03673 0.00367129
1.04301 0.00379621
1.04929 0.00392497
1.05558 0.00405766
1.06186 0.00419441
1.06814 0.0043353
1.07442 0.00448047
1.08071 0.00463002
1.08699 0.00478406
1.09327 0.00494272
1.09956 0.00510611
1.10584 0.00527437
1.11212 0.00544761
1.11841 0.00562597
1.12469 0.00580958
1.13097 0.00599856
1.13726 0.00619307
1.14354 0.00639324
1.14982 0.00659921
1.15611 0.00681112
1.16239 0.00702913
1.16867 0.00725339
1.17496 0.00748405
1.18124 0.00772126
1.18752 0.0079652
1.19381 0.00821602
1.20009 0.00847388
1.20637 0.00873897
1.21265 0.00901144
1.21894 0.00929148
1.22522 0.00957926
1.2315 0.00987498
1.23779 0.0101788
1.24407 0.0104909
1.25035 0.0108115
1.25664 0.0111408
1.26292 0.011479
1.2692 0.0118263
1.27549 0.0121829
1.28177 0.012549
1.28805 0.0129247
1.29434 0.0133105
1.30062 0.0137063
1.3069 0.0141125
1.31319 0.0145294
1.31947 0.014957
1.32575 0.0153957
1.33204 0.0158457
1.33832 0.0163072
1.3446 0.0167805
1.35088 0.0172659
1.35717 0.0177634
1.36345 0.0182736
1.36973 0.0187965
1.37602 0.0193324
1.3823 0.0198816
1.38858 0.0204444
1.39487 0.0210211
1.40115 0.0216119
1.40743 0.022217
1.41372 0.0228369
1.42 0.0234717
1.42628 0.0241217
1.43257 0.0247873
1.43885 0.0254687
1.44513 0.0261662
1.45142 0.0268802
1.4577 0.0276109
1.46398 0.0283587
1.47027 0.0291238
1.47655 0.0299066
1.48283 0.0307073
1.48911 0.0315264
1.4954 0.0323641
1.50168 0.0332207
1.50796 0.0340966
1.51425 0.0349922
1.52053 0.0359076
1.52681 0.0368434
1.5331 0.0377998
1.53938 0.0387771
1.54566 0.0397758
1.55195 0.0407961
1.55823 0.0418384
1.56451 0.0429031
1.5708 0.0439905
1.57708 0.0451009
1.58336 0.0462348
1.58965 0.0473925
1.59593 0.0485743
1.60221 0.0497806
1.6085 0.0510118
1.61478 0.0522683
1.62106 0.0535503
1.62734 0.0548584
1.63363 0.0561927
1.63991 0.0575538
1.64619 0.058942
1.65248 0.0603577
1.65876 0.0618012
1.66504 0.0632729
1.67133 0.0647732
1.67761 0.0663025
1.68389 0.0678611
1.69018 0.0694494
1.69646 0.0710678
1.70274 0.0727166
1.70903 0.0743963
1.71531 0.0761072
1.72159 0.0778497
1.72788 0.0796241
1.73416 0.0814309
1.74044 0.0832703
1.74673 0.0851428
1.75301 0.0870487
1.75929 0.0889884
1.76558 0.0909622
1.77186 0.0929706
1.77814 0.0950138
1.78442 0.0970922
1.79071 0.0992062
1.79699 0.101356
1.80327 0.103542
1.80956 0.105765
1.81584 0.108025
1.82212 0.110322
1.82841 0.112656
1.83469 0.115028
1.84097 0.117439
1.84726 0.119888
1.85354 0.122377
1.85982 0.124904
1.86611 0.12747
1.87239 0.130077
1.87867 0.132723
1.88496 0.13541
1.89124 0.138138
1.89752 0.140906
1.90381 0.143716
1.91009 0.146566
1.91637 0.149459
1.92265 0.152393
1.92894 0.155369
1.93522 0.158388
1.9415 0.161449
1.94779 0.164553
1.95407 0.1677
1.96035 0.17089
1.96664 0.174123
1.97292 0.1774
1.9792 0.180721
1.98549 0.184085
1.99177 0.187493
1.99805 0.190945
2.00434 0.194441
2.01062 0.197981
2.0169 0.201566
2.02319 0.205195
2.02947 0.208869
2.03575 0.212587
2.04204 0.216349
2.04832 0.220157
2.0546 0.224008
2.06088 0.227905
2.06717 0.231846
2.07345 0.235832
2.07973 0.239862
2.08602 0.243937
2.0923 0.248056
2.09858 0.25222
2.10487 0.256427
2.11115 0.260679
2.11743 0.264975
2.12372 0.269315
2.13 0.273699
2.13628 0.278126
2.14257 0.282596
2.14885 0.28711
2.15513 0.291667
2.16142 0.296266
2.1677 0.300908
2.17398 0.305592
2.18027 0.310318
2.18655 0.315085
2.19283 0.319894
2.19911 0.324744
2.2054 0.329634
2.21168 0.334564
2.21796 0.339534
2.22425 0.344544
2.23053 0.349593
2.23681 0.35468
2.2431 0.359805
2.24938 0.364968
2.25566 0.370168
2.26195 0.375404
2.26823 0.380677
2.27451 0.385985
2.2808 0.391327
2.28708 0.396704
2.29336 0.402115
2.29965 0.407559
2.30593 0.413035
2.31221 0.418543
2.3185 0.424083
2.32478 0.429652
2.33106 0.435251
2.33734 0.440879
2.34363 0.446535
2.34991 0.452219
2.35619 0.457929
2.36248 0.463664
2.36876 0.469425
2.37504 0.47521
2.38133 0.481018
2.38761 0.486848
2.39389 0.4927
2.40018 0.498572
2.40646 0.504463
2.41274 0.510374
2.41903 0.516302
2.42531 0.522246
2.43159 0.528206
2.43788 0.534181
2.44416 0.54017
2.45044 0.546171
2.45673 0.552183
2.46301 0.558206
2.46929 0.564238
2.47558 0.570278
2.48186 0.576325
2.48814 0.582379
2.49442 0.588436
2.50071 0.594498
2.50699 0.600562
2.51327 0.606627
2.51956 0.612692
2.52584 0.618756
2.53212 0.624817
2.53841 0.630875
2.54469 0.636928
2.55097 0.642974
2.55726 0.649013
2.56354 0.655044
2.56982 0.661064
2.57611 0.667073
2.58239 0.673069
2.58867 0.679051
2.59496 0.685018
2.60124 0.690968
2.60752 0.6969
2.61381 0.702813
2.62009 0.708706
2.62637 0.714576
2.63265 0.720422
2.63894 0.726244
2.64522 0.73204
2.6515 0.737809
2.65779 0.743548
2.66407 0.749257
2.67035 0.754935
2.67664 0.760579
2.68292 0.766189
2.6892 0.771763
2.69549 0.777301
2.70177 0.782799
2.70805 0.788258
2.71434 0.793675
2.72062 0.79905
2.7269 0.804381
2.73319 0.809666
2.73947 0.814904
2.74575 0.820095
2.75204 0.825236
2.75832 0.830326
2.7646 0.835364
2.77088 0.840349
2.77717 0.845278
2.78345 0.850152
2.78973 0.854968
2.79602 0.859726
2.8023 0.864424
2.80858 0.86906
2.81487 0.873634
2.82115 0.878144
2.82743 0.88259
2.83372 0.886969
2.84 0.891281
2.84628 0.895524
2.85257 0.899697
2.85885 0.9038
2.86513 0.90783
2.87142 0.911787
2.8777 0.91567
2.88398 0.919477
2.89027 0.923208
2.89655 0.926862
2.90283 0.930437
2.90911 0.933932
2.9154 0.937347
2.92168 0.94068
2.92796 0.94393
2.93425 0.947097
2.94053 0.95018
2.94681 0.953178
2.9531 0.956089
2.95938 0.958913
2.96566 0.96165
2.97195 0.964298
2.97823 0.966856
2.98451 0.969324
2.9908 0.971702
2.99708 0.973988
3.00336 0.976181
3.00965 0.978282
3.01593 0.98029
3.02221 0.982203
3.0285 0.984021
3.03478 0.985745
3.04106 0.987372
3.04734 0.988904
3.05363 0.990339
3.05991 0.991676
3.06619 0.992917
3.07248 0.994059
3.07876 0.995103
3.08504 0.996049
3.09133 0.996896
3.09761 0.997644
3.10389 0.998292
3.11018 0.998841
3.11646 0.999291
3.12274 0.999641
3.12903 0.999891
3.13531 1.00004
3.14159 1.00009
3.14788 1.00004
3.15416 0.999891
3.16044 0.999641
3.16673 0.999291
3.17301 0.998841
3.17929 0.998292
3.18557 0.997644
3.19186 0.996896
3.19814 0.996049
3.20442 0.995103
3.21071 0.994059
3.21699 0.992917
3.22327 0.991676
3.22956 0.990339
3.23584 0.988904
3.24212 0.987372
3.24841 0.985745
3.25469 0.984021
3.26097 0.982203
3.26726 0.98029
3.27354 0.978282
3.27982 0.976181
3.28611 0.973988
3.29239 0.971702
3.29867 0.969324
3.30496 0.966856
3.31124 0.964298
3.31752 0.96165
3.32381 0.958913
3.33009 0.956089
3.33637 0.953178
3.34265 0.95018
3.34894 0.947097
3.35522 0.94393
3.3615 0.94068
3.36779 0.937347
3.37407 0.933932
3.38035 0.930437
3.38664 0.926862
3.39292 0.923208
3.3992 0.919477
3.40549 0.91567
3.41177 0.911787
3.41805 0.90783
3.42434 0.9038
3.43062 0.899697
3.4369 0.895524
3.44319 0.891281
3.44947 0.886969
3.45575 0.88259
3.46204 0.878144
3.46832 0.873634
3.4746 0.86906
3.48088 0.864424
3.48717 0.859726
3.49345 0.854968
3.49973 0.850152
3.50602 0.845278
3.5123 0.840349
3.51858 0.835364
3.52487 0.830326
3.53115 0.825236
3.53743 0.820095
3.54372 0.814904
3.55 0.809666
3.55628 0.804381
3.56257 0.79905
3.56885 0.793675
3.57513 0.788258
3.58142 0.782799
3.5877 0.777301
3.59398 0.771763
3.60027 0.766189
3.60655 0.760579
3.61283 0.754935
3.61911 0.749257
3.6254 0.743548
3.63168 0.737809
3.63796 0.73204
3.64425 0.726244
3.65053 0.720422
3.65681 0.714576
3.6631 0.708706
3.66938 0.702813
3.67566 0.6969
3.68195 0.690968
3.68823 0.685018
3.69451 0.679051
3.7008 0.673069
3.70708 0.667073
3.71336 0.661064
3.71965 0.655044
3.72593 0.649013
3.73221 0.642974
3.7385 0.636928
3.74478 0.630875
3.75106 0.624817
3.75734 0.618756
3.76363 0.612692
3.76991 0.606627
3.77619 0.600562
3.78248 0.594498
3.78876 0.588436
3.79504 0.582379
3.80133 0.576325
3.80761 0.570278
3.81389 0.564238
3.82018 0.558206
3.82646 0.552183
3.83274 0.546171
3.83903 0.54017
3.84531 0.534181
3.85159 0.528206
3.85788 0.522246
3.86416 0.516302
3.87044 0.510374
3.87673 0.504463
3.88301 0.498572
3.88929 0.4927
3.89557 0.486848
3.90186 0.481018
3.90814 0.47521
3.91442 0.469425
3.92071 0.463664
3.92699 0.457929
3.93327 0.452219
3.93956 0.446535
3.94584 0.440879
3.95212 0.435251
3.95841 0.429652
3.96469 0.424083
3.97097 0.418543
3.97726 0.413035
3.98354 0.407559
3.98982 0.402115
3.99611 0.396704
4.00239 0.391327
4.00867 0.385985
4.01496 0.380677
4.02124 0.375404
4.02752 0.370168
4.0338 0.364968
4.04009 0.359805
4.04637 0.35468
4.05265 0.349593
4.05894 0.344544
4.06522 0.339534
4.0715 0.334564
4.07779 0.329634
4.08407 0.324744
4.09035 0.319894
4.09664 0.315085
4.10292 0.310318
4.1092 0.305592
4.11549 0.300908
4.12177 0.296266
4.12805 0.291667
4.13434 0.28711
4.14062 0.282596
4.1469 0.278126
4.15319 0.273699
4.15947 0.269315
4.16575 0.264975
4.17204 0.260679
4.17832 0.256427
4.1846 0.25222
4.19088 0.248056
4.19717 0.243937
4.20345 0.239862
4.20973 0.235832
4.21602 0.231846
4.2223 0.227905
4.22858 0.224008
4.23487 0.220157
4.24115 0.216349
4.24743 0.212587
4.25372 0.208869
4.26 0.205195
4.26628 0.201566
4.27257 0.197981
4.27885 0.194441
4.28513 0.190945
4.29142 0.187493
4.2977 0.184085
4.30398 0.180721
4.31027 0.1774
4.31655 0.174123
4.32283 0.17089
4.32911 0.1677
4.3354 0.164553
4.34168 0.161449
4.34796 0.158388
4.35425 0.155369
4.36053 0.152393
4.36681 0.149459
4.3731 0.146566
4.37938 0.143716
4.38566 0.140906
4.39195 0.138138
4.39823 0.13541
4.40451 0.132723
4.4108 0.130077
4.41708 0.12747
4.42336 0.124904
4.42965 0.122377
4.43593 0.119888
4.44221 0.117439
4.4485 0.115028
4.45478 0.112656
4.46106 0.110322
4.46734 0.108025
4.47363 0.105765
4.47991 0.103542
4.48619 0.101356
4.49248 0.0992062
4.49876 0.0970922
4.50504 0.0950138
4.51133 0.0929706
4.51761 0.0909622
4.52389 0.0889884
4.53018 0.0870487
4.53646 0.0851428
4.54274 0.0832703
4.54903 0.0814309
4.55531 0.0796241
4.56159 0.0778497
4.56788 0.0761072
4.57416 0.0743963
4.58044 0.0727166
4.58673 0.0710678
4.59301 0.0694494
4.59929 0.0678611
4.60557 0.0663025
4.61186 0.0647732
4.61814 0.0632729
4.62442 0.0618012
4.63071 0.0603577
4.63699 0.058942
4.64327 0.0575538
4.64956 0.0561927
4.65584 0.0548584
4.66212 0.0535503
4.66841 0.0522683
4.67469 0.0510118
4.68097 0.0497806
4.68726 0.0485743
4.69354 0.0473925
4.69982 0.0462348
4.70611 0.0451009
4.71239 0.0439905
4.71867 0.0429031
4.72496 0.0418384
4.73124 0.0407961
4.73752 0.0397758
4.7438 0.0387771
4.75009 0.0377998
4.75637 0.0368434
4.76265 0.0359076
4.76894 0.0349922
4.77522 0.0340966
4.7815 0.0332207
4.78779 0.0323641
4.79407 0.0315264
4.80035 0.0307073
4.80664 0.0299066
4.81292 0.0291238
4.8192 0.0283587
4.82549 0.0276109
4.83177 0.0268802
4.83805 0.0261662
4.84434 0.0254687
4.85062 0.0247873
4.8569 0.0241217
4.86319 0.0234717
4.86947 0.0228369
4.87575 0.022217
4.88203 0.0216119
4.88832 0.0210211
4.8946 0.0204444
4.90088 0.0198816
4.90717 0.0193324
4.91345 0.0187965
4.91973 0.0182736
4.92602 0.0177634
4.9323 0.0172659
4.93858 0.0167805
4.94487 0.0163072
4.95115 0.0158457
4.95743 0.0153957
4.96372 0.014957
4.97 0.0145294
4.97628 0.0141125
4.98257 0.0137063
4.98885 0.0133105
4.99513 0.0129247
5.00142 0.012549
5.0077 0.0121829
5.01398 0.0118263
5.02027 0.011479
5.02655 0.0111408
5.03283 0.0108115
5.03911 0.0104909
5.0454 0.0101788
5.05168 0.00987498
5.05796 0.00957926
5.06425 0.00929148
5.07053 0.00901144
5.07681 0.00873897
5.0831 0.00847388
5.08938 0.00821602
5.09566 0.0079652
5.10195 0.00772126
5.10823 0.00748405
5.11451 0.00725339
5.1208 0.00702913
5.12708 0.00681112
5.13336 0.00659921
5.13965 0.00639324
5.14593 0.00619307
5.15221 0.00599856
5.1585 0.00580958
5.16478 0.00562597
5.17106 0.00544761
5.17734 0.00527437
5.18363 0.00510611
5.18991 0.00494272
5.19619 0.00478406
5.20248 0.00463002
5.20876 0.00448047
5.21504 0.0043353
5.22133 0.00419441
5.22761 0.00405766
5.23389 0.00392497
5.24018 0.00379621
5.24646 0.00367129
5.25274 0.00355011
5.25903 0.00343256
5.26531 0.00331855
5.27159 0.00320798
5.27788 0.00310076
5.28416 0.0029968
5.29044 0.00289601
5.29673 0.0027983
5.30301 0.00270359
5.30929 0.00261179
5.31557 0.00252283
5.32186 0.00243662
5.32814 0.00235309
5.33442 0.00227217
5.34071 0.00219377
5.34699 0.00211784
5.35327 0.00204429
5.35956 0.00197306
5.36584 0.00190408
5.37212 0.00183729
5.37841 0.00177263
5.38469 0.00171003
5.39097 0.00164943
5.39726 0.00159078
5.40354 0.00153401
5.40982 0.00147908
5.41611 0.00142593
5.42239 0.0013745
5.42867 0.00132474
5.43496 0.00127661
5.44124 0.00123005
5.44752 0.00118502
5.4538 0.00114146
5.46009 0.00109935
5.46637 0.00105863
5.47265 0.00101925
5.47894 0.000981185
5.48522 0.000944387
5.4915 0.000908816
5.49779 0.000874436
5.50407 0.000841208
5.51035 0.000809097
5.51664 0.000778068
5.52292 0.000748087
5.5292 0.000719121
5.53549 0.000691136
5.54177 0.000664103
5.54805 0.00063799
5.55434 0.000612769
5.56062 0.00058841
5.5669 0.000564885
5.57319 0.000542168
5.57947 0.000520233
5.58575 0.000499053
5.59203 0.000478604
5.59832 0.000458862
5.6046 0.000439804
5.61088 0.000421407
5.61717 0.00040365
5.62345 0.00038651
5.62973 0.000369968
5.63602 0.000354003
5.6423 0.000338596
5.64858 0.000323729
5.65487 0.000309382
5.66115 0.000295539
5.66743 0.000282182
5.67372 0.000269295
5.68 0.000256861
5.68628 0.000244865
5.69257 0.000233293
5.69885 0.000222128
5.70513 0.000211358
5.71142 0.000200969
5.7177 0.000190948
5.72398 0.000181281
5.73027 0.000171956
5.73655 0.000162961
5.74283 0.000154286
5.74911 0.000145918
5.7554 0.000137846
5.76168 0.000130061
5.76796 0.000122552
5.77425 0.000115309
5.78053 0.000108324
5.78681 0.000101586
5.7931 9.50866e-05
5.79938 8.8818e-05
5.80566 8.27716e-05
5.81195 7.69396e-05
5.81823 7.13141e-05
5.82451 6.58879e-05
5.8308 6.06538e-05
5.83708 5.56049e-05
5.84336 5.07346e-05
5.84965 4.60365e-05
5.85593 4.15045e-05
5.86221 3.71325e-05
5.8685 3.29149e-05
5.87478 2.88461e-05
5.88106 2.49209e-05
5.88734 2.1134e-05
5.89363 1.74807e-05
5.89991 1.39561e-05
5.90619 1.05557e-05
5.91248 7.27505e-06
5.91876 4.11001e-06
5.92504 1.05648e-06
5.93133 -1.88942e-06
5.93761 -4.73146e-06
5.94389 -7.47323e-06
5.95018 -1.01182e-05
5.95646 -1.26697e-05
5.96274 -1.51308e-05
5.96903 -1.75047e-05
5.97531 -1.97943e-05
5.98159 -2.20024e-05
5.98788 -2.41317e-05
5.99416 -2.61848e-05
6.00044 -2.81641e-05
6.00673 -3.0072e-05
6.01301 -3.19109e-05
6.01929 -3.36827e-05
6.02557 -3.53898e-05
6.03186 -3.70339e-05
6.03814 -3.8617e-05
6.04442 -4.0141e-05
6.05071 -4.16075e-05
6.05699 -4.30182e-05
6.06327 -4.43747e-05
6.06956 -4.56784e-05
6.07584 -4.69309e-05
6.08212 -4.81334e-05
6.08841 -4.92873e-05
6.09469 -5.03937e-05
6.10097 -5.14539e-05
6.10726 -5.2469e-05
6.11354 -5.344e-05
6.11982 -5.43679e-05
6.12611 -5.52537e-05
6.13239 -5.60982e-05
6.13867 -5.69023e-05
6.14496 -5.76668e-05
6.15124 -5.83925e-05
6.15752 -5.908e-05
6.1638 -5.973e-05
6.17009 -6.03432e-05
6.17637 -6.09201e-05
6.18265 -6.14613e-05
6.18894 -6.19672e-05
6.19522 -6.24384e-05
6.2015 -6.28753e-05
6.20779 -6.32783e-05
6.21407 -6.36477e-05
6.22035 -6.39839e-05
6.22664 -6.42871e-05
6.23292 -6.45578e-05
6.2392 -6.4796e-05
6.24549 -6.50021e-05
6.25177 -6.51761e-05
6.25805 -6.53183e-05
6.26434 -6.54287e-05
6.27062 -6.55076e-05
6.2769 -6.55548e-05
