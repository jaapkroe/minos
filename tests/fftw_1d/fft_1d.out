0          0.2507            0
1         -0.2058     2.22e-18
2          0.1138            0
3        -0.04242   -8.882e-18
4         0.01065    1.277e-17
5       -0.001803    1.318e-18
6       0.0002055     -6.8e-18
7       -1.59e-05   -6.257e-18
8        7.46e-07   -1.409e-20
9      -1.313e-07   -5.175e-18
